runelite-client/src/main/java/net/runelite/client/plugins/kingdomofmiscellania/KingdomPlugin.java

```diff
@@ -122,20 +189,60 @@ private void removeKingdomInfobox()
 		}
 	}
 
+	private int estimateCoffer(long lastChangedEpoch, int lastCoffer)
+	{
+		int daysSince = Math.max((int) (Instant.now().getEpochSecond() - lastChangedEpoch) / 86_400, 0);
```

Comment:
Don't use a magic number to represent one day's worth of seconds. Make a `private static final int` constant instead and represent it with `(int) Duration.of(1, ChronoUnit.DAYS).get(ChronoUnit.SECONDS)`